target/linux/x86/patches-5.15/120-usb-net-for-fm350.patch

```diff
@@ -0,0 +1,59 @@
+Author: Ji Li <liji@huachen.link>
+Date:   Thu Jun 23 16:56:44 2022 +0800
+
+    USB: serial: option: add MEDIATEK FM350-GL module support
```

Comment:
Please send this patch also to upstream Linux. OpenWrt is fine to take it already before it is integrated in upstream, but we prefer patches to be inside of upstream Linux, then OpenWrt does not have to maintain them forever. 
There are some internal process at Intel to help with Linux upstreaming, this change looks pretty simple so I do not think it will be hard. There was an internal mailing list at Intel for such reviews, maybe Andy Shevchenko ( andriy.shevchenko@linux.intel.com ) can help you with that.